mod-orders-storage/schemas/reason_for_closure.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Reason for closure object",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "UUID identifying this reason for closure",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "reasonForClosure": {
+      "description": "Reason for closure",
+      "type": "string"
+    },
+    "isSystem": {
```

Comment:
I think we might want to use an enum here instead... e.g.

`"source": [ "User", "System" ]`

mod-orders would always use the "User" source, overriding whatever the user sends.

Then in mod-orders-storage we'd load have the System records loaded when a tenant enables the module.  We just did something similar in mod-invoice-storage for batch-groups.  It looks like the System records currently live in the UI code (not great).